pom.xml

```diff
@@ -307,7 +307,7 @@
                   <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
                   <Bundle-Name>${project.name}</Bundle-Name>
                   <Bundle-SymbolicName>${bundle-symbolic-name}.source</Bundle-SymbolicName>
-                  <Bundle-Vendor>Eclipse.org - Jetty</Bundle-Vendor>
+                  <Bundle-Vendor>Eclipse Jetty Project</Bundle-Vendor>
```

Comment:
Yeah, it probably should be called "Eclipse Jetty Project".
WDYT @jmcc0nn3ll  ?

Reply:
That is what it is in most of our jars, so let's be consistent